code-pushup.config.ts

```diff
@@ -87,18 +87,24 @@ const config: CoreConfig = {
     {
       slug: 'bug-prevention',
       title: 'Bug prevention',
+      description:
+        'Bug prevention category consists of rules and checks that are essential for stability and functionality of your codebase. It currently contains [ESLint](https://eslint.org/) rules that indicate potential bugs in your codebase.',
```

Comment:
```suggestion
      description: 'Lint rules that find potential bugs in your code.',
```

Reply:
I wondered about this. Shouldn't we include the general idea of a category to showcase as well? Although on this project it doesn't matter as much, I suppose.